website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+- [dbt Cloud features](/docs/develop/dbt-cloud-features)
+- [Version control basics](/docs/collaborate/git/version-control-basics)
+- [dbt Commands](/docs/reference/dbt-commands)
+
+
+## Related questions
+
+<details>
+  <summary>Is there a cost to using the Cloud IDE?</summary>
+  <div>
+    <div>Not at all! You can use dbt Cloud when you sign up for the <a href="https://www.getdbt.com/pricing/">Free Developer plan</a>, which comes with one developer seat. If you’d like to access more features or have more developer seats, you can upgrade your account to the Team or Enterprise plan. See <a href="https://www.getdbt.com/pricing/">dbt pricing plans</a> for more details. </div>
+  </div>
+</details>
+<details>
+  <summary>Can I be a contributor to dbt Cloud?</summary>
+  <div>
+    <div>Anyone can contribute to the dbt project. And whether it's a dbt package, a plugin, dbt-core, or this documentation site, contributing to the open source code that supports the dbt ecosystem is a great way to level yourself up as a developer, and give back to the community. See <a href="https://docs.getdbt.com/docs/contributing/oss-expectations">Contributing</a> for details on what to expect when contributing to the dbt open source software (OSS). </div>
+  </div>
+</details>
+<details>
+  <summary>What is the difference between developing on the Cloud IDE and on the CLI?</summary>
+  <div>
+    <div>There are two main ways to develop with dbt: using the web-based IDE in dbt Cloud or using the command-line interface (CLI) in dbt Core: <br></br>
+      <span>&mdash;</span>
+      <b>dbt Cloud IDE</b> dbt Cloud is a web-based application that allows you to develop dbt projects with the IDE, includes a purpose-built scheduler, and provides an easier way to share your dbt documentation with your team. The IDE is a faster and more reliable way to deploy your dbt models and provides a real-time editing and execution environment for your dbt project. <br></br>
+      <span>&mdash;</span>
+      <b>dbt Core CLI</b> The command line interface (CLI) uses <a href="https://docs.getdbt.com/docs/introduction">dbt Core</a>, an <a href="https://github.com/dbt-labs/dbt">open-source</a> software that’s freely available. You can build your dbt project in a code editor, like Jetbrains or VSCode, and run dbt commands from the command line.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>What type of support is provided with dbt Cloud?</summary>
+  <div>
+    <div>The global dbt Support team is available to help dbt Cloud users by email or in-product live chat. Developer and Team accounts offer 24x5 support, while Enterprise customers have priority access and options for custom coverage. <br></br> If you have project-related or modeling questions, review <a href="https://docs.getdbt.com/docs/dbt-cloud/cloud-dbt-cloud-support">our Support page</a> or <a href="http://getdbt.slack.com/">dbt Community Slack</a> to get help as well. </div>
+  </div>
+</details>
```

Comment:
i love how these expandables look! 

looking at the content, i think several of them can just be a link in Related docs like the Cloud support (/docs/dbt-support) expandable and a couple others. note that we've got git issue #2085 to refresh that support page